docs/operating-scylla/procedures/cluster-management/arbiter-dc.rst

```diff
@@ -0,0 +1,36 @@
+=======================================================
+Preventing Quorum Loss in Multi-DC Clusters
```

Comment:
Here, too, the title is too broad.
Arbiters can prevent Quorum Loss in some use cases but not all.
For example, it will **not** help in the following cases:
- 1 DC
- 3 or more DCs
- 2 asymmetric DCs

Reply:
> Arbiters can prevent Quorum Loss in some use cases but not all. For example, it will **not** help in the following cases:
> 
>     * 1 DC
> 
>     * 3 or more DCs
> 
>     * 2 asymmetric DCs

That's true if the Arbiter DC contains one node. It could help in the case of at least two DCs if it contained sufficiently many zero-token nodes. Examples:
- DC1: 5 nodes, DC2: 3 nodes: -- Arbiter with 3 zero-token nodes,
- DC1: 10 nodes, DC2: 4 nodes, DC3: 2 nodes -- Arbiter with 5 zero-token nodes.

So, the Arbiter DC can help in such cases but we don't want to use it in such cases. The reason is that we plan to do #18793 in 6.3. AFAIK, fixing this issue will:
- solve the case with at least 3 DCs entirely (all DCs will have the same number of voters, so no DC will have at least half of the voters),
- reduce the 2 asymmetric DCs case to the 2 symmetric DCs case (both DCs will have the same number of voters, so the Arbiter DC with one voting zero-token node will be sufficient).

I suppose we will update the documentation to cover all possible Multi-DC setups when we fix #18793. For now, we conveniently ignore them. In the end, we will only need Arbiters with one zero-token node to cover all setups.